modules/system/opposed-wfrp4e.js

```diff
@@ -115,9 +115,14 @@ export default class OpposedWFRP {
 
   renderOpposedStart() {
     return new Promise(async resolve => {
-      let attacker = WFRP_Utility.getToken(this.attackerTest.context.speaker) || this.attacker.prototypeToken;
+      let attacker = game.canvas.tokens.get(this.attackerTest.context.cardOptions.speaker.token)?.document ?? this.attacker.prototypeToken;
       let defender
```

Comment:
why wasn't `getToken` working here?

Reply:
![image](https://user-images.githubusercontent.com/7930626/200287592-b453134e-3bbe-478a-8507-432edf61898a.png)


Scene was null for speaker so it was always falling back to token prototype